src/core/Accordion/types.ts

```diff
@@ -26,6 +26,13 @@ export type AccordionData = {
    * @param index - The index of the clicked accordion item.
    */
   onClick?: (index: number) => void;
+
+  /**
+   * Indicates whether the accordion item is interactive.
+   * When false, the item cannot be expanded or collapsed by user interaction.
+   * @default true
+   */
+  interactive?: boolean;
```

Comment:
This prop allows static rows to be added to the accordion menu, as in, ones with no content that don't expand